samples/bluetooth/direction_finding_central/sysbuild.cmake

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(NOT SB_CONFIG_SOC_NRF5340_CPUAPP)
+  if(NOT "bt-ll-sw-split" IN_LIST ${DEFAULT_IMAGE}_SNIPPET)
+    list(APPEND ${DEFAULT_IMAGE}_SNIPPET ${SNIPPET})
+    list(APPEND ${DEFAULT_IMAGE}_SNIPPET bt-ll-sw-split)
+    list(REMOVE_DUPLICATES ${DEFAULT_IMAGE}_SNIPPET)
```

Comment:
I'm wondering about this remove of duplicates. If there is no `bt-ll-sw-split` then there is no reason of calling this. 
What more as far as I know the order of snippets in SNIPPET variable matters. So I would not do a remove operation but force user to apply snippet once in right order.

Reply:
@nordicjm , what is your opinion on this?

Reply:
was to prevent a snippet being added multiple times by cache but seems it is not an issue, can be removed